#
# The YAML file is a lightweight markup language file that supports all
# ROS parameter types. It's used by rosparam to load static parameters to
# the Parameter Server, which are commonly used to initialize part of a
# system and rarely change on run-time. Sometimes the user modifies this
# file before starting a node. This file is usually in the /config subdirectory
# of a package. Note that when dynamic reconfiguration is implemented for
# a package, most parameters will be dynamic, making this file barely used.
#
#                    _________ NOTES _________
#
# If 'single' mode is used, only port and operation are necessary, so the
# other parameters should be commented.
# 
# More details on "ScienceMode - RehaStim Stimulation Device.pdf" 
# document from Thomas Shauer, Nils-Otto Negaard and Carsten Behling
#

port: /dev/ttyUSB0  # Port address for stimulator
operation: ccl  # Operation modes (single/ccl)

channel_stim: [1,2,3,4,5,6,7,8]  # List of active channels
channel_lf: []  # List of low freq channels (values must be in channel_stim)
n_factor: 0  # Number from 0 to 7 of skipped periods in low freq channels
freq: 50  # Main stimulation frequency in Hz (this overrides ts1)
ts1: 15  # Main stimulation period in ms (1-1024.5 ms in 0.5 steps)
ts2: 5  # Inter-pulse time in ms (1.5-17 ms in 0.5 steps)